app/views/admin/announcements/_announcement.html.slim

```diff
@@ -1,4 +1,4 @@
-div.announcement class=time_period_class(announcement)
+= div_for(announcement, class: ['announcement', time_period_class(announcement)])
```

Comment:
Look at the example in the Rails documentation. It adds the class, unless you're saying the documentation is wrong of which then you should submit a PR to have that fixed :)